src/generator/operations.ts

```diff
@@ -118,7 +122,7 @@ function formatHeaderResponseValue(propName: string, header: string, schema: Sch
     text += `\t\t\t${respObj}.Metadata[hh[len(prefix):]] = resp.Header.Get(hh)\n`;
     text += '\t\t}\n';
     text += '\t}\n';
-    return text;
+    return [text, true];
```

Comment:
Please consider returning an object instead with well-named fields to describe the return value.

Reply:
Also, consider adding unit tests to test the scenario you are improving in this PR.

Reply:
I'm not sure about adding an object for this specific case, since it will only ever be used for this function? And probably wont be useful elsewhere, but please let me know if you think it is better to include. 

The test for this specific fix is the fact that the code compiles. Without this fix the code doesnt compile and CI would not pass. This is why I updated the storage blobs api version so that this can be tested.